index.css

```diff
@@ -0,0 +1,138 @@
+.page {
+    background-color: darkcyan;
+}
+
+.topBar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color:cadetblue;
+    height: 100px;
+    padding: 20px 40px 20px 40px;
```

Comment:
Instead of writing out each of the paddings individually here you can use the shorthand we learned about in lecture because your top = bottom and your left = right. So instead this could be replaced with "padding: 20px 40px;" where the first value is the padding for your top & bottom and the second value is the padding for your left & right. 
This is only a small improvement but it can save you from writing a lot of repeat values in the future!